velox/exec/tests/utils/QueryAssertions.h

```diff
@@ -59,6 +63,14 @@ class DuckDbQueryRunner {
     return allRows;
   }
 
+  // returns the DuckDB TPC-H Extension Query as string for a given 'queryNo'
```

Comment:
Comments should start with a capital letter and end with a period. E.g. "Returns the ..."